core/chains/cardano.go

```diff
@@ -19,41 +19,41 @@ type CardanoResponse struct {
 }
 
 func Cardano() (int, error) {
-    url := "https://www.balanceanalytics.io/api/mavdata.json"
+	url := "https://www.balanceanalytics.io/api/mavdata.json"
```

Comment:
@BrazyDevelopment please revert changes in the cardano file as the pull request has nothing to do with cardano